doc/how-to/use-external-builder-vm.md

```diff
@@ -0,0 +1,17 @@
+# How to use external builder vm
+
+This guide will cover how to use an external OpenStack builder VM to build and snapshot images.
+
+### Command
+
+Run the following command to create a snapshot image of a VM launched by the 
+github-runner-image-builder.
+
+To find out what flavor is available, run `openstack list flavor`.
```

Comment:
Nitpick: To find out what flavour is available 

I do understand that the command is American spelling and you might like to remain consistent with the command, but I still think the spelling outside of the commands should be changed to British.